assignment5/assignment5.js

```diff
@@ -0,0 +1,27 @@
+const fs = require('fs');
+
+function readingFile(file) {
+  return new Promise(function (resolve, reject) {
+    fs.readFile(file, 'utf8' , function (err, data) {
+      if (err) {
+        reject(err);
```

Comment:
If in case Error occurs, will your program stop the execution of further steps???